tools/invoke-mermaid.sh

```diff
@@ -2,11 +2,12 @@
 set -eo pipefail
 
 cfg_file=$(dirname "$0")/mermaid.json
+cfg_p_file=$(dirname "$0")/mermaid-p.json
 
 # This script is meant to be run in parallel with itself, so it uses --no to
 # disable the prompt to install new packages (and avoid npx racing with itself).
 # Use tools/install-dependencies.sh to install the package explicitly.
-npx --no -- @mermaid-js/mermaid-cli@9.1.4 --backgroundColor black --configFile "$cfg_file" "$@"
+npx --no -- @mermaid-js/mermaid-cli@9.1.4 -p "$cfg_p_file" --backgroundColor black --configFile "$cfg_file" "$@"
```

Comment:
nit: similar to above: `--puppeteerConfigFile="$cfg_p_file"`

Reply:
Much better, thank you!